src/index.ts

```diff
@@ -61,6 +61,12 @@ app.use(
       return;
     }
 
+    if (req.session) {
+      // TODO: Abort transaction is async
```

Comment:
Rule of thumb I've been using with westegg now since we have multiple people working on it is to not add any more todos but instead just open a ticket for it. If the line is specific and you want to make sure you remember the context then link it in the issue you make.

Reply:
The PR is still in progress. I left that in there so I would remember to discuss it during our work session today

Reply:
Next time I'll prefix it with Draft: or something so it doesn't accidentally get merged

Reply:
Use DNM for (Do no merge) in the PR title. That way it can be for drafts or if something is wrong.